src/configuration/GlobalConfiguration.h

```diff
@@ -23,6 +23,8 @@ class GlobalConfiguration
 public:
     static void print();
 
+    static const double LEAKY_RELU_SLOPE;
```

Comment:
Is it a single slope for all leaky relus in the query? Because the python interface suggested each could have a separate slope.

Reply:
I agree that we should allow each leaky relu to have their own slope.